src/styles/Notification.css

```
@@ -0,0 +1,61 @@
+  font-size: 14px;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 10px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.3);
+  animation: slideIn 0.3s ease-out forwards;
+  pointer-events: all; /* Allow interaction with close button */
+}
+
+@keyframes slideIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)